src/components/ModalMinimap/index.js

```diff
@@ -3,20 +3,21 @@ import PropTypes from 'prop-types'
 import Modal from 'react-modal'
 import MaterialIcon from '../MaterialIcon'
 import Minimap from '../Minimap'
+import { t, Trans, Select } from '@lingui/macro'
 import './styles.scss'
 
-const minimapAboutText = <>
+const minimapAboutText = <Trans comment='The about section for minimaps.'>
   <p>Jump to a part of the collection containing matches by clicking on an active square.</p>
   <p>In the minimap diagram, each square represents part of this collection.
   Colored squares represent parts that contain one or more matches for your search.</p>
-</>
+</Trans>
```

Comment:
Do you have a preference for me replacing empty divs with the Trans macro? In these spots, it seemed to make sense to not have an empty div around the Trans macro.

Reply:
Agree - the only reason those empty divs are there is to satisfy React.